torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1074,8 +1074,10 @@ RegisterOperators reg(
            const auto type = node->output()->type()->expect<ClassType>();
            const size_t numAttrs = type->numAttributes();
            return [type, numAttrs](Stack& stack) {
+             auto cu = type->compilation_unit().lock();
```

Comment:
StrongTypePtr and the cu lookup should go _outside_ the closure to prevent recompute every time.